src/Tempest/Database/DatabaseConfig.php

```diff
@@ -7,15 +7,35 @@
 final class DatabaseConfig
 {
     public function __construct(
-        public readonly DatabaseDriver $driver,
-        public array $migrations = [],
+        private readonly DatabaseDriver $driver,
```

Comment:
We make config properties public. 

In fact, we're actually gonna wait for PHP 8.4 until we release a v1 of Tempest, and we're going to refactor all interfaces to use property hooks instead of methods. It's not related to these properties, but I figured I'd let you know anyway :)

Reply:
Sounds interesting 😋 

for now I'll make it public.
```suggestion
        public readonly DatabaseDriver $driver,
```